### PR TITLE
[Emotion L4] Media Player

### PR DESCRIPTION
**Pull request by ryanmccombe**
Resolves #NUMBER

**Overall change:** _A very high-level summary of easily-reproducible changes that can be understood by non-devs._

**Code changes:**

- _A bullet point list of key code changes that have been made._
- _When describing code changes, try to communicate **how** and **why** you implemented something a specific way, not just **what** has changed._

---

- [ ] I have assigned myself to this PR and the corresponding issues
- [ ] Automated jest tests added (for new features) or updated (for existing features)
- [ ] This PR requires manual testing

### CODE REVIEW

File: packages/components/psammead-media-player/src/Placeholder/__snapshots__/index.test.jsx.snap [hunk 1]
```diff
@@ -538,31 +769,6 @@ exports[`Media Player: Placeholder should render an audio placeholder without du
 
 exports[`Media Player: Placeholder should render no-js styles when noJsClassName prop is used 1`] = `
 .emotion-6 {
-  background-color: #222222;
```
Comment (simonsinclair) [inline packages/components/psammead-media-player/src/Placeholder/__snapshots__/index.test.jsx.snap:541]: Hmm, potential loss of styles from here? Seems to be no-js related.

File: packages/components/psammead-media-player/src/__snapshots__/index.test.jsx.snap [hunk 2]
```diff
@@ -472,32 +701,13 @@ exports[`Media Player: Canonical Entry renders the audio skin 1`] = `
 `;
 
 exports[`Media Player: Canonical Entry renders with no-js styles when noJsClassName prop is used 1`] = `
-.emotion-6 {
-  background-color: #222222;
```
Comment (simonsinclair) [inline packages/components/psammead-media-player/src/__snapshots__/index.test.jsx.snap:476]: The same block here.

**Review by RichardPK — APPROVED** [review]

File: packages/components/psammead-media-player/src/Placeholder/__snapshots__/index.test.jsx.snap (hunk 1 above)
Comment (simonsinclair) [reply packages/components/psammead-media-player/src/Placeholder/__snapshots__/index.test.jsx.snap:541]: False alarm.

File: packages/components/psammead-media-player/src/__snapshots__/index.test.jsx.snap (hunk 2 above)
Comment (simonsinclair) [reply packages/components/psammead-media-player/src/__snapshots__/index.test.jsx.snap:476]: As above.

**Review by simonsinclair — APPROVED** [review]

**Pull request merged**